### DISCUSSION TITLE
Adding new sessions via the plus button

### DISCUSSION THREAD

**Discussion by maartsen** [1Remote/1Remote#500] [Ideas]
Hi,

The following use case:
* All my RDP sessions are configured to go full screen ("New session always use full screen").
* But sometimes I've got multiple sessions open and I want to group them
* So I click on the window button and I get a toolbar with the session and a plus button
* If I press the plus button, I can start a new RDP session, but that will go full screen (as per settings).

My change request is, can the plus button ignore the full screen and dock directly in the tool bar?

Best regards,
Michel

**Comment by VShawn** [comment]
Unfortunately, the toolbar in fullscreen mode is provided by Microsoft, and we cannot customize it. Therefore, we cannot pin the plus button to the toolbar.
